import-page/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "id": "!!REPLACE THIS WITH YOUR PACKAGE ID!!",
```

Comment:
!!

Reply:
This is how we want to have it so that the users know where to insert their package ID. We don't want to publish a real package ID/this is how the other examples had it.

Reply:
Gotcha, that makes sense